documentation/mkdocs.yml

```diff
@@ -32,6 +33,7 @@ nav:
         - pipelineStashFiles: steps/pipelineStashFiles.md
         - pipelineStashFilesAfterBuild: steps/pipelineStashFilesAfterBuild.md
         - pipelineStashFilesBeforeBuild: steps/pipelineStashFilesBeforeBuild.md
+        - piperPipelineStagePost: steps/piperPipelineStagePost.md
```

Comment:
let's remove this part for now and focus on kaniko ...

Reply:
... mmmhhh. Clearly my fault. The pull request is too big. I should have opened to PRs, one for each file ;-)

What is the reason for not publishing docu for the `piperPipeineStagePost`? Is this still work in progress? My concern is: we are getting warnings from mkdocs in case there is a file not referenced from here. Without additional knowledge about the internals here is it hard to judge if it is 'fine for now' to have that warning.

Reply:
done, in this PR we have only karniko